app/models/payment.rb

```diff
@@ -0,0 +1,20 @@
+class Payment < ApplicationRecord
+  belongs_to :user
+  belongs_to :company, optional: true # used for branding_fee
+
+  validates_presence_of :user, :payment_type, :status
+
+  ORDER_PAYMENT_STATUS = {
```

Comment:
What is the purpose of this Hash? It seems that you're doing some sort of mapping from the HIPS terminology to SHF, but I don't quite understand what or why.  I'm unclear because the terms "order" and "payment" are too ambiguous; those terms can apply to either/both systems.

A comment here would probably help me a lot.

Reply:
Please see added comment.

I thought that naming the hash `HIPS_ORDER_STATUS_TO_SHF_PAYMENT_STATUS` would be a bit unwieldy.  :)